lib/core/src/server/loadCustomBabelConfig.js

```diff
@@ -43,24 +45,25 @@ function loadFromPath(babelConfigPath) {
   return config;
 }
 
-export default function(configDir, defaultConfig) {
-  let babelConfig = loadFromPath(path.resolve(configDir, '.babelrc'));
-  let inConfigDir = true;
+function isBabelLoader8() {
+  // eslint-disable-next-line import/no-dynamic-require,global-require
+  const babelLoaderPkg = require(resolveSync('babel-loader/package.json', {
+    basedir: process.cwd(),
+  }));
+  return satisfies(babelLoaderPkg.version, '>=8.0.0-0');
+}
 
-  if (!babelConfig) {
-    babelConfig = loadFromPath('.babelrc');
-    inConfigDir = false;
-  }
+export default function(configDir, defaultConfig) {
+  const babelConfig = loadFromPath(path.resolve(configDir, '.babelrc'));
 
   if (babelConfig) {
     // If the custom config uses babel's `extends` clause, then replace it with
     // an absolute path. `extends` will not work unless we do this.
     if (babelConfig.extends) {
-      babelConfig.extends = inConfigDir
-        ? path.resolve(configDir, babelConfig.extends)
-        : path.resolve(babelConfig.extends);
+      babelConfig.extends = path.resolve(configDir, babelConfig.extends);
     }
+    return babelConfig;
   }
 
-  return babelConfig || defaultConfig;
+  return isBabelLoader8() ? defaultConfig : {};
```

Comment:
So, our default babel config is for babel8. After removing `babelrc: false`, can't we just return the default?

Reply:
Ah, just understood my mistake 🤦‍♂️